### PR TITLE
Possible docs typo

### PR DESCRIPTION
**Pull request by AlexisRenchon**
Possible typo in the docs

## Content

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by Sbozzolo** [comment]
Thank you :)

I'll merge this once you caught all the other typos

File: docs/src/user_guide.md [hunk 1]
```diff
@@ -92,12 +91,12 @@ Let us examine what is in a `ScheduledDiagnostic` in more details:
...
   and output (`compute_schedule_func` and `output_schedule_func`). We have two
   separate entries one for compute and one for output because we might want to
-  control them separately. For instance, we might want to take the average of
+  control them separately. For example, we might want to take the average of
   something every 10 steps, and output it the average every 100 iterations.
   `schedule` functions are powerful, so there is an entire section dedicated to
   them below. `compute_schedule_func` and `output_schedule_func` are likely
-  going to the same unless there are temporal reductions.
-- an `output_writer`, an object that knows how what to do with the output.
+  going to the same (? same what ? output ?) unless there are temporal reductions.
```
Comment (Sbozzolo) [inline docs/src/user_guide.md:98]:
```suggestion
  going to be the same unless there are temporal reductions.
```

File: docs/src/user_guide.md [hunk 2]
```diff
@@ -137,7 +136,7 @@ function compute_if_larger_than100(integrator)
 end
 ```
 
-Strictly speaking, schedules do not have to functions, but callable objects. For
+Strictly speaking, schedules do not have to (? be ?) functions, but callable objects. For
```
Comment (Sbozzolo) [inline docs/src/user_guide.md:139]:
```suggestion
Strictly speaking, schedules do not have to be functions, but callable objects. For
```

**Review by Sbozzolo — APPROVED** [review]
Thank you! Feel free to merge once you incorporate the fixes below and squash the commits.

File: docs/src/developer_guide.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 This page provides additional documentation on abstractions to use
 `ClimaDiagnostics`. Before reading this page, make sure you are familiar with
-the terminology. You know to know what a [`DiagnosticVariable`](@ref ClimaDiagnostics.DiagnosticVariables.DiagnosticVariable) and a
+the terminology. You need to know what a [`DiagnosticVariable`](@ref ClimaDiagnostics.DiagnosticVariables.DiagnosticVariable) and a
 [`ScheduledDiagnostic`](@ref ClimaDiagnostics.ScheduledDiagnostics.ScheduledDiagnostic) are.
 
 There are three components needed to add support for `ClimaDiagnostics.jl` in your package.
```
Comment (AlexisRenchon) [inline docs/src/developer_guide.md:8]: @Sbozzolo Here, we say "three components", but we then only list two. (or maybe I am missing something?) - I think maybe 1. should be DiagnosticVariable?

**Pull request merged**